### PR TITLE
UI: Changes navigation url from home page to previous page 

### PR DESCRIPTION
**Pull request by pratap0007**
-    Changes navigation url from home page to previous page url
      - Previously navigation url was home page after successful authentication now it will be previous page url
      - Removes force reload
 -    Fixes to reload the images of Footer and Background components
       - Images of Footer and Background components doesn't reload with same uri
          so this patch adds global Date() as param in image  url to reload it
    

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: ui/src/components/Background/index.tsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import { Text, TextContent, TextVariants } from '@patternfly/react-core';
+import background from '../../assets/logo/background.jpg';
```
Comment (PuneetPunamiya) [inline ui/src/components/Background/index.tsx:3]: nit: 
```suggestion
import Background from '../../assets/logo/background.jpg';
```

**Review by PuneetPunamiya — COMMENTED** [review]
Thanks for the pr :call_me_hand: 

Just one thing `package-lock.json`  gets updated could you please check that once

**Comment by vinamra28** [comment]
> Thanks for the pr 
> 
> Just one thing `package-lock.json` gets updated could you please check that once

For me it's not changing

**Pull request merged**